platform/mellanox/asic_table.j2

```diff
@@ -60,7 +60,9 @@
             "cell_size": "96",
             "pipeline_latency": "19",
             "mac_phy_delay": "0.8",
-            "peer_response_time": "3.8"
+            "peer_response_time": "3.8",
+            "port_reserved_shp": "20",
+            "port_max_shp": "8192"
         },
```

Comment:
Can this PR be merged without https://github.com/sonic-net/sonic-swss/pull/3272??

Reply:
Yes
Swss [#3272](https://github.com/sonic-net/sonic-swss/pull/3272) depends on this PR